docs/content/preview/migrate/manual-import/import-data.md

```diff
@@ -13,28 +13,92 @@ menu:
 type: docs
 ---
 
-The next step is to import the PostgreSQL data into YugabyteDB.
+{{< tip title="Migrate using YugabyteDB Voyager" >}}
+To automate your migration from PostgreSQL to YugabyteDB, use [YugabyteDB Voyager](../../yb-voyager/). To learn more, refer to the [import schema](../../yb-voyager/migrate-steps/#import-schema) and [import data](../../yb-voyager/migrate-steps/#import-data) steps.
+{{< /tip >}}
 
-{{< note title="Note" >}}
-After the data import step, remember to recreate any constraints and triggers that might have been disabled to speed up loading the data. This would ensure that the database will perform relational integrity checking for data going forward.
-{{< /note >}}
+## Import data from CSV files
+
+To import data that was previously exported into CSV files, use the COPY FROM command as follows:
+
+```sql
+COPY <table_name>
+    FROM ‘<table_name>.csv’
+    WITH (
+        FORMAT CSV DELIMITER ',' HEADER
+        ROWS_PER_TRANSACTION 1000
+        DISABLE_FK_CHECK
+    );
+```
+
+In the command above, the `ROWS_PER_TRANSACTION` parameter splits the load into smaller transactions (1000 rows each in this example), instead of running a single transaction spawning across all the data in the file. Additionally, the `DISABLE_FK_CHECK` parameter skips the foreign key checks for the duration of the import process.
+
+Both `ROWS_PER_TRANSACTION` and `DISABLE_FK_CHECK` parameters are recommended for the initial import of the data, especially for large tables, because they significantly reduce the total time required to import the data. You can import multiple files in a single COPY command to further speed up the process. Following is a sample example:
+
+```sql
```

Comment:
Either break into separate input and output code blocks, or just tag the whole thing as `output.sql`